.git2gus/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "productTag": "a1aEE000000GLQHYA4",
+    "defaultBuild": "Backlog",
+    "hideWorkItemUrl": true
+}
```

Comment:
do you want to add `"statusWhenClosed": "FIXED"`?

Reply:
done